templates/mentorship_relation_accepted.html

```diff
@@ -0,0 +1,11 @@
+<p>Hi {{ request_sender }},</p>
+<br>
+<p>Congratulations!</p>
+<br>
+<p>Your mentorship request has been accepted by {{ request_receiver }}.
+    He will be your {{ role }} until {{ end_date }}.
```

Comment:
```suggestion
    {{ request_receiver }} will be your {{ role }} until {{ end_date }}.
```
We should not use such pronouns within the app. We don't know the mentor/mentee, the user's gender.

Reply:
Updated the change!